### ISSUE TITLE
Bato.to: Covers do not load while browsing

### ISSUE THREAD

**Issue by vipzzx** [tachiyomiorg/tachiyomi-extensions#13882]
### Source information and language

Bato.to 1.3.30 All

### Steps to reproduce

1. Open Bato.to extension
2. Wait for cover art to load
3. A lot will not load unless you go into webpage and go back to app.

### Expected behavior

Show all cover arts

### Actual behavior

Not showing cover art.
![Screenshot_20221016-162138_TachiyomiJ2K](https://user-images.githubusercontent.com/59638889/196056669-2cb92329-b7a1-45f4-93b7-e4224b5c12af.jpg)

### Tachiyomi version

J2K 1.5.9

### Android version

12

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by Clouddark75** [comment]
Hmm, they load fine here. Can you add more to it?

**Comment by vipzzx** [comment]
Well now that's super awkward. Seems to be working fine now. It has been an issue for a few weeks already. Guess it magically got fixed 🤷🏻‍♀️

**Comment by XenonFenix** [comment]
I was having the same issue for about a month. It happened when I changed phones. 

What fixed it for me was:
1. create a backup
2. remove all of the manga from the library
3. Go to Settings > Advanced > Clear database (I also cleared everything else like cache and covers)
4. Restore backup
5. If any of the covers still didn't load, take note of the name, chapter, and source (ex. Bato) and remove it from library
6. Go to the source and search for the manga and the cover load

Hope this helps anyone who comes across this issue in the future.
